packages/lib-classifier/src/components/Classifier/components/TaskArea/TaskArea.js

```diff
@@ -24,17 +24,37 @@ function storeMapper (stores) {
 }
 
 class TaskArea extends React.Component {
+  constructor() {
+    super()
+
+    this.state = {
+      activeIndex: 0
+    }
+  }
+
   onTabClick (activeIndex) {
     const { setActiveTutorial, tutorial } = this.props
     if (activeIndex === 1) setActiveTutorial(tutorial)
     if (activeIndex === 0) setActiveTutorial()
+    this.setActiveIndex(activeIndex)
+  }
+
+  onClose () {
+    const { setActiveTutorial } = this.props
+    setActiveTutorial()
+    this.setActiveIndex(0)
+  }
+
+  setActiveIndex(activeIndex) {
+    this.setState({ activeIndex })
   }
 
   render () {
     const { disableTutorialTab } = this.props
 
     return (
       <Tabs
+        activeIndex={this.state.activeIndex}
         className={this.props.className}
```

Comment:
Not related to this PR, but I don't think there's a `className` prop to pass as we're not wrapping anything in a styled component: https://www.styled-components.com/docs/advanced#styling-normal-react-components

Reply:
Might be cruft from a previous version of the tabs?